src/java.base/share/classes/java/lang/Class.java

```diff
@@ -423,11 +424,10 @@ private static Class<?> forName(String className, Class<?> caller)
     /**
      * Returns the {@code Class} object associated with the class or
      * interface with the given string name, using the given class loader.
-     * Given the fully qualified name for a class or interface (in the same
-     * format returned by {@code getName}) this method attempts to
-     * locate and load the class or interface.  The specified class
-     * loader is used to load the class or interface.  If the parameter
-     * {@code loader} is null, the class is loaded through the bootstrap
+     * Given the {@linkplain ##binary-name binary name} for a class or interface,
```

Comment:
Is double hash a mistake?

Reply:
That’s a new **Javadoc** feature available since **JDK 20**: [JDK‑8200337] ([GH‑10395])

[JDK‑8200337]: https://bugs.openjdk.org/browse/JDK-8200337 "[JDK‑8200337] Generalize see and link tags for user‑defined anchors"
[GH‑10395]: https://github.com/openjdk/jdk/pull/10395